### DISCUSSION TITLE
Apakah bisa mengirim pesan dengan menggunakan variable tambahan?

### DISCUSSION THREAD

**Discussion by ky0shir089** [kirimwa-id/forum#180] [Tanya Jawab]
apakah ini bisa kirim pesan dengan variable tambahan yang di setup sendiri?

misal "selamat pagi {name}" atau {var1} {var2} dst...

terima kasih

**Comment by udien-arrow** [comment]
semakin sepi pak...

**Comment by rioastamal** [comment]
@ky0shir089 Tidak. Fitur seperti itu bisa diimplementasikan di sisi developer sendiri bukan di API.
